flatpak/org.mamedev.MAME.yaml

```diff
@@ -0,0 +1,71 @@
+id: org.mamedev.MAME
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: mame
+separate-locales: false
+finish-args:
+  - --device=all
+  - --filesystem=home
+  - --filesystem=host:ro
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --allow=bluetooth
+modules:
+  - name: mame
+    buildsystem: simple
+    build-options:
```

Comment:
The autotools buildsystem is not possible?

Reply:
No